apps/mediacenter/style/index.css

```diff
@@ -72,7 +72,7 @@ body {
 }
 
 #search-panel sl-input::part(base) {
-  border-radius: var(--sl-border-radius-large);
+  border-radius: 1em;
```

Comment:
let's keep using shoelace variables: `--sl-border-radius-x-large` should be what you want.

Reply:
alright